components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2102,6 +2112,29 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
         await this.guardAccess({ kind: "costCenter", /*subject: costCenter,*/ owner }, operation);
     }
+
+    protected mapBilledSession(s: usage.BilledSession): BillableSession {
+        function mandatory<T>(s: T, m: (s: T) => string = (s) => "" + s): string {
+            if (!s) {
+                throw new Error("empty value in usage.BilledSession");
```

Comment:
Would be nice to also return which Instance ID had empty value, for easier debugging